res/poketool/waza/meson.build

```diff
@@ -1,5 +1,30 @@
+subdir('pl_waza_tbl')
+
+pl_waza_tbl_files = custom_target('pl_waza_tbl_files',
+    output: 'pl_waza_tbl_files',
```

Comment:
The value of `output` should match the names of the files that actually get generated by the command, or else meson will always rerun the latter, because it can't find the build artifacts.

Reply:
I refactored the build script to generate the targets sequentially rather than build all the binaries at once. The new code in `res/poketool/waza/pl_waza_tbl/meson.build` is:

```
foreach src_file : pl_waza_tbl_srcs
    if i < 10
        bin_name = '000' + i.to_string() + '.bin'
    elif i < 100
        bin_name = '00' + i.to_string() + '.bin'
    elif i < 1000
        bin_name = '0' + i.to_string() + '.bin'
    else
        bin_name = i.to_string() + '.bin'
    endif

    pl_waza_tbl_bins += custom_target(bin_name,
        output: bin_name,
        input: files(src_file),
        command: [
            movedata_py,
            '-i', '@INPUT@',
            '-o', '@OUTDIR@'
        ]
    )

    i += 1
endforeach
```

I modeled this after how the `pl_msg` narc is built: https://github.com/pret/pokeplatinum/blob/9af28d6d28434d51dad12ead44fd18560a7d0ab7/res/msgdata/pl_msg/meson.build#L730